src/Options.php

```diff
@@ -686,6 +686,24 @@ public function setTracesSampler(?callable $sampler): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the amount of memory to reserve in case of a fatal error.
+     * The number is in bytes. Null indicated the default should be used.
+     */
+    public function getFatalReservedMemory(): ?int
+    {
+        return $this->options['fatal_error_reserved_memory'];
```

Comment:
Doesn't makes more sense to call this option `fatal_error_handler_reserved_memory_amount`? I don't like it being so long, but the memory is clearly affecting the error handler

Reply:
Absolutely. I wasn't sure about the long name as well. I don't have a better name. Change incoming.